src/Types/IDEntityPts.php

```diff
@@ -45,9 +53,13 @@ public static function normalize($value): ?string
      */
     public function isValid(): bool
     {
-        /** @var PtsCodeValidatorExtension $validator */
-        $validator = static::getContainer()->make(PtsCodeValidatorExtension::class);
+        if (\is_string($this->value) && $this->value !== '') {
+            /** @var PtsCodeValidatorExtension $validator */
+            $validator = static::getContainer()->make(PtsCodeValidatorExtension::class);
+
+            return \is_string($this->value) && $validator->passes('', $this->value);
```

Comment:
Redundant `\is_string($this->value)`. We are check this condition above.

Reply:
Fixed